### PR TITLE
cooling_effect

### PR DESCRIPTION
**Pull request by Lu-Whale**
cooling_effect implemented

### CODE REVIEW

File: src/models/cooling_effect.js [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+ *
+ * Note: The activity as well as the air speed modify the insulation characteristics
+ * of the clothing and the adjacent air layer. Consequently, the ISO 7730 states that
+ * the clothing insulation shall be corrected {@link #ref_2|[2]}. The ASHRAE 55 Standard corrects
+ * for the effect of the body movement for met equal or higher than 1.2 met using
+ * the equation clo = Icl × (0.6 + 0.4/met) The dynamic clothing insulation, clo,
+ * can be calculated using the function `clo_dynamic` which is in .utilities.js.
+ *
+ * @param {number} [wme=0] - external work
+ * @param {'SI','IP'} [units= "SI"] - select the SI (International System of Units) or the IP (Imperial Units) system.
```
Comment (szaldivar) [inline src/models/cooling_effect.js:40]:
```suggestion
 * @param {"SI" | "IP"} [units= "SI"] - select the SI (International System of Units) or the IP (Imperial Units) system.
```

File: src/models/cooling_effect.js [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+        false,
+        {
+          round: false,
+          calculate_ce: true,
+        },
+      ) - initial_set_tmp
+    );
+  }
+
+  function findRoot(func, a, b) {
```
Comment (szaldivar) [inline src/models/cooling_effect.js:116]: declare this function outside this function and take a callback for `func`

**Review by szaldivar — CHANGES_REQUESTED** [review]

File: src/models/cooling_effect.js (hunk 1 above)
Comment (Lu-Whale) [reply src/models/cooling_effect.js]: done

File: src/models/cooling_effect.js (hunk 2 above)
Comment (Lu-Whale) [reply src/models/cooling_effect.js]: done

**Review by szaldivar — APPROVED** [review]

**Pull request merged**